container/docker/factory.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/google/cadvisor/fs"
 	info "github.com/google/cadvisor/info/v1"
 
-	"github.com/fsouza/go-dockerclient"
+	docker "github.com/fsouza/go-dockerclient"
```

Comment:
It's not necessary, but I think it makes it easier to parse the code (and figure out what `docker` refers to). We do it in some other places, but I can revert it if you prefer.